src/firebase-app.ts

```diff
@@ -423,4 +423,39 @@ export class FirebaseApp {
       );
     }
   }
+
+  private useConfigVar_(): FirebaseAppOptions {
+    if (this.options_.databaseURL !== undefined &&
+      this.options_.projectId !== undefined &&
+      this.options_.storageBucket !== undefined) {
+      return;
+    }
+    if (process.env[this.firebaseInternals_.CONFIG_FILE_VAR]) {
+      let contents;
+      try {
+        contents = fs.readFileSync(process.env[this.firebaseInternals_.CONFIG_FILE_VAR], 'utf8');
+      } catch (ignored) {
```

Comment:
When will this return an error? Does it make sense to expose this to the developer?

Reply:
throwing